src/models/code_language.py

```diff
@@ -0,0 +1,20 @@
+"""
+Model for coding language
+"""
+
+from abc import ABC
+
+
+# ABC = Abstract Base Class
+class LanguageStorable(ABC):
+    pass
+
+
+class CodeLanguage(LanguageStorable):
+    def __init__(self, dictionary):
+        if isinstance(dictionary, dict):
```

Comment:
I don't want you to get in the habit of checking to make sure everything passed into a constructor is what it should be.  That's a bit non-pythonic - I'd like to see you maybe follow the pattern of type hinting the parameter and then just assuming the person who uses the constructor will use it responsibly.